velox/functions/sparksql/String.h

```diff
@@ -1130,4 +1130,53 @@ struct ReplaceFunction {
   }
 };
 
+template <typename T>
+struct FindInSetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int32_t>& result,
+      const arg_type<Varchar>& str,
+      const arg_type<Varchar>& strArray) {
+    int32_t index = 1;
+    int32_t lastComma = -1;
+
+    auto strString = std::string_view(str);
```

Comment:
Maybe, better to rename it, as well as `strArrayString`. Thanks!

Reply:
Ok, The strString will be renamed to `match` and the strArrayString is `arrayStr`. Do you think arrayStr is good ?Thanks.